src/fsharp/BinaryResourceFormats.fsi

```diff
@@ -1,21 +1,21 @@
 ﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
 
-module internal FSharp.Compiler.BinaryResourceFormats 
+module internal FSharp.Compiler.BinaryResourceFormats
 
 open FSharp.Compiler.AbstractIL.IL
 
-module VersionResourceFormat = 
+module VersionResourceFormat =
 
-    val VS_VERSION_INFO_RESOURCE: 
+    val VS_VERSION_INFO_RESOURCE:
         (ILVersionInfo * ILVersionInfo * int32 * int32 * int32 * int32 * int32 * int64) *
         seq<string * #seq<string * string>> *
-        seq<int32 * int32>
-          -> byte[]
-        
+        seq<int32 * int32> ->
+            byte []
```

Comment:
Looks like another place worth fixing (if not yet 🙂)

Reply:
What's wrong here? Did you catch up on https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-types-in-signatures?

Reply:
It was about the added space.

Reply:
The space between the `byte` and `[]` has been resolved. Available in the v5 alphas.
The space before `byte` is part of the indentation of the return type.

Reply:
Cool, thanks for confirming!